### PR TITLE
Updates for 2021

### PR DESCRIPTION
**Pull request by zcollins0**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,15 +7,16 @@
  - [x] Tribunal TGIF
  - [ ] Hiking
  - [x] The Social network x1
- - [ ] Brazilian Steakhouse (for a good reason)
+ - [ ] <s>Brazilian Steakhouse (for a good reason)</s>
```
Comment (domfarolino) [inline README.md:10]: Due to vegan restrictions?

File: README.md [hunk 2]
```diff
@@ -7,15 +7,16 @@
...
  - [ ] Kings Island
- - [ ] 16 Bit
+ - [ ] <s>16 Bit</s>
```
Comment (domfarolino) [inline README.md:12]: Permanently closed?

File: README.md [hunk 3]
```diff
@@ -7,15 +7,16 @@
...
  - [ ] <s>Dueling Piano Bar </s>
  - [ ] Nada
  - [ ] Short Vine Bar Hop - Hit every bar on shortvine from Kroger to Taste of Belgium
  - [x] Engineering Luau
  - [x] Buzzed Bull Creamery
  - [x] Ohio State Fair
  - [ ] Canoeing
- - [ ] Terry's Turf Club
+ - [ ] <s>Terry's Turf Club</s>
```
Comment (domfarolino) [inline README.md]: ??

File: README.md (hunk 1 above)
Comment (zcollins0) [reply README.md:10]: Yeah, vegan plus Laura is vegetarian.

File: README.md (hunk 2 above)
Comment (zcollins0) [reply README.md:12]: Yep, 16 Bit no longer exists.

File: README.md (hunk 3 above)
Comment (zcollins0) [reply README.md]: Terry’s Turf Club is canceled due to sexual harassment and not paying employees, or something like that.

Comment (kurtlewis) [reply README.md]: Looks like it's under new ownership and just "Turf Club". Not sure if it makes it worth not crossing out - we're pretty unlikely to do it. https://www.bizjournals.com/cincinnati/news/2019/01/09/new-owner-talks-about-what-will-change-stay-the.html

File: README.md [hunk 4]
```diff
@@ -7,15 +7,16 @@
...
  - [ ] Downtown street car barhop
+ - [ ] Napa/Sonoma Meetup
```
Comment (kurtlewis) [inline README.md]: I'm curious why you wrote Sonoma?

Comment (zcollins0) [reply README.md]: It’s nearby and also has wine. Will switch to just Napa.

File: README.md (hunk 3 above)
Comment (zcollins0) [reply README.md]: Updated.

**Review by domfarolino — APPROVED** [review]

**Review by ltebben — APPROVED** [review]
LGTM!

**Pull request merged**